lib/cisco_node_utils/bridge_domain.rb

```diff
@@ -142,12 +148,17 @@ def default_bd_name
       config_get_default('bridge_domain', 'bd_name')
     end
 
-    # Bridge-Domain type change to fabric-control
-    # bridge-domain 100
-    #   fabric-control
     # This type property can be defined only for one bd
     def fabric_control
-      config_get('bridge_domain', 'fabric_control', bd: @bd_ids)
+      match = config_get('bridge_domain', 'fabric_control', bd: @bd_ids)
```

Comment:
You could probably also simplify this using structured output but I will let you work out the details.

Reply:
done